index.js

```diff
@@ -321,6 +303,86 @@ async function updatePlatformaticComment (octokit, commentId, comment) {
   })
 }
 
+function getApplicationPackageName (appType) {
+  return `@platformatic/${appType}`
+}
+
+async function checkPlatformaticDependency (projectPath) {
+  const packageJsonPath = join(projectPath, 'package.json')
+
+  const packageJsonExist = await isFileAccessible(packageJsonPath)
+  if (!packageJsonExist) return
+
+  const packageJsonData = await readFile(packageJsonPath, 'utf8')
+  const packageJson = JSON.parse(packageJsonData)
+
+  const dependencies = packageJson.dependencies
+  if (
+    dependencies !== undefined &&
+    dependencies.platformatic !== undefined
+  ) {
+    core.warning('Move platformatic dependency to devDependencies to speed up deployment')
+  }
+}
+
+async function getLatestPackageVersion (packageName) {
+  const { statusCode, body } = await request(
+    `https://registry.npmjs.org/${packageName}`,
+    {
+      method: 'GET'
+    }
+  )
+
+  if (statusCode !== 200) {
+    const error = body.text()
+    throw new Error(`Cannot get latest version of platformatic package ${statusCode} ${error}`)
+  }
+
+  const packageInfo = await body.json()
+  return packageInfo['dist-tags'].latest
+}
+
+async function installPackage (packageName, version) {
+  return new Promise((resolve, reject) => {
+    core.info(`Installing ${packageName}@v${version} package...`)
+    exec(`npm install ${packageName}@v${version}`, { cwd: __dirname }, (error) => {
+      if (error) {
+        core.info('Failed to install platformatic dependency')
+        reject(error)
+      } else {
+        core.info('Successfully installed platformatic dependency')
+        resolve()
+      }
+    })
+  })
+}
+
+async function getPackagePath (pathToProject, packageName) {
+  const latestPackageVersion = await getLatestPackageVersion(packageName)
+  core.info(`Latest version of ${packageName} is ${latestPackageVersion}`)
+
+  const localPackagePath = join(pathToProject, 'node_modules', packageName)
+  const localPackageInstalled = await isFileAccessible(localPackagePath)
+
+  if (localPackageInstalled) {
+    const localPackageVersion = await getPackageVersion(localPackagePath)
+    core.info(`Local version of ${packageName} is ${localPackageVersion}`)
+
+    if (localPackageVersion === latestPackageVersion) return localPackagePath
+  }
+
```

Comment:
If the local version is not the latest, let's print a warning:

"In our beta phase, we only support using the latest release of Platformatic".

Reply:
Should we throw if versions are not equal or warning would be enough?

Reply:
warning is enough for now